cmd/apprepository-controller/pkg/apis/apprepository/v1alpha1/types.go

```diff
@@ -47,12 +47,11 @@ type AppRepositorySpec struct {
 	DockerRegistrySecrets []string `json:"dockerRegistrySecrets,omitempty"`
 	// In case of an OCI type, the list of repositories is needed
 	// as there is no API for the index
-	// DEPRECATED: Use FilterRules instead
 	OCIRepositories []string `json:"ociRepositories,omitempty"`
 	// TLSInsecureSkipVerify skips TLS verification
 	TLSInsecureSkipVerify bool `json:"tlsInsecureSkipVerify,omitempty"`
 	// FilterRule allows to filter packages based on a JQuery
-	FilterRule FilterRuleSpec `json:"filterRules,omitempty"`
+	FilterRule FilterRuleSpec `json:"filterRule,omitempty"`
```

Comment:
API docs nitpicky guy here :P reminder: we should also update the openapi accordingly

Reply:
true, I think this field is not even documented. I am not sure how to work with the new approach, can you document how to modify the API docs?

Reply:
Sure, I'll do it as soon as I finish the pinniped thing. Thanks for the suggestion.

Briefly, in the meantime, it is just a matter of accessing  https://app.swaggerhub.com/apis/kubeapps/Kubeapps/, modifying the field in the yaml view, click on save, click on sync, click on push and the changes will appear on the `openapi` branch. Then you can just send the PR and that's all.